src/Zicht/Bundle/MessagesBundle/Command/TranslateCommand.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * @copyright Zicht Online <http://zicht.nl>
+ */
+
+namespace Zicht\Bundle\MessagesBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\Translation\Loader\ArrayLoader;
+use Symfony\Component\Translation\Util\XliffUtils;
+use Symfony\Component\Yaml\Yaml;
+use Zicht\Bundle\MessagesBundle\Translator\BatchTranslatorInterface;
+
+class TranslateCommand extends Command
+{
+    /** @var string */
+    protected static $defaultName = 'zicht:messages:translate';
+
+    /** @var  BatchTranslatorInterface */
+    private $batchTranslator;
+
+    /**
+     * @param BatchTranslatorInterface $batchTranslator
+     */
+    public function setBatchTranslator(BatchTranslatorInterface $batchTranslator)
+    {
+        $this->batchTranslator = $batchTranslator;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->addArgument('file', InputArgument::REQUIRED)
+            ->addOption('--source', '-s', InputOption::VALUE_REQUIRED, 'The source-language.')
+            ->addOption('--target', '-t', InputOption::VALUE_OPTIONAL, 'The target-language. Override if auto-discovered target is not in line with your translation API specs.');
+    }
+
```

Comment:
Misschien nog een `initialize()` method toevoegen die checkt op de aanwezigheid van een `BatchTranslatorInterface` in `$this->batchTranslator` en anders een exception gooit?

Reply:
Toegevoegd.